models/DataObject/Service.php

```diff
@@ -552,20 +374,6 @@ public static function calculateCellValue(AbstractObject $object, array $helperD
         });
     }
 
-    public static function getHelperDefinitions(): array
```

Comment:
BC break here.

Reply:
Thank you! 

Resolved by https://github.com/pimcore/pimcore/pull/16745/commits/351fb2a09a5bb589b2ef016705aaefbaaa1578ae and the upgrade note